pkg/operator/edgefs/cluster/target/layout.go

```diff
@@ -56,7 +56,7 @@ func getIdDevLinkName(dls string) (dl string) {
 	dlsArr := strings.Split(dls, " ")
 	for i := range dlsArr {
 		s := strings.Replace(dlsArr[i], "/dev/disk/by-id/", "", 1)
-		if strings.Contains(s, "/") || strings.Contains(s, "wwn-") {
+		if strings.Contains(s, "/") || strings.Contains(s, "wwn-") || strings.Contains(s, "nvme-nvme.") || strings.Contains(s, "nvme-eui.") {
```

Comment:
the devices specified by fullPath won't be skipped by the check for `/`?

Reply:
done